keyboards/crkbd/keymaps/thunderbird2086/config.h

```diff
@@ -32,63 +33,80 @@
 #define RGBLIGHT_SLEEP
 
 #if defined(RGBLIGHT_ENABLE)
+#    define RGBLIGHT_LAYERS
 #    define RGBLIGHT_LIMIT_VAL 150
 #    define RGBLIGHT_HUE_STEP 16
 #    define RGBLIGHT_SAT_STEP 32
 #    define RGBLIGHT_VAL_STEP 32
 
+#    define RGBLIGHT_EFFECT_ALTERING
+#    define RGBLIGHT_EFFECT_BREATHING
+#    define RGBLIGHT_EFFECT_CHRISTMAS
+#    define RGBLIGHT_EFFECT_KNIGHT
 #    define RGBLIGHT_EFFECT_RAINBOW_MOOD
 #    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
-#    define RGBLIGHT_EFFECT_KNIGHT
+#    define RGBLIGHT_EFFECT_RGB_TEST
+#    define RGBLIGHT_EFFECT_SNAKE
+#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+#    define RGBLIGHT_EFFECT_TWINKLE
 #endif
 
 #if defined(RGB_MATRIX_ENABLE)
-#    define RGB_MATRIX_KEYPRESSES               // reacts to keypresses
-#    define RGB_DISABLE_WHEN_USB_SUSPENDED      // turn off effects when suspended
-#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 150
+#   define SPLIT_LAYER_STATE_ENABLE
+#   define RGB_MATRIX_KEYPRESSES                // reacts to keypresses
+#   define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#   define RGB_DISABLE_WHEN_USB_SUSPENDED       // turn off effects when suspended
+#   define RGB_MATRIX_MAXIMUM_BRIGHTNESS 150
         // limits maximum brightness of LEDs to 150 out of 255. Higher may cause the controller to crash.
-#    define RGB_MATRIX_HUE_STEP 32
-#    define RGB_MATRIX_SAT_STEP 64
-#    define RGB_MATRIX_VAL_STEP 64
-#    define RGB_MATRIX_SPD_STEP 20
-#    define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_COLOR
+#   define RGB_MATRIX_HUE_STEP 32
+#   define RGB_MATRIX_SAT_STEP 64
+#   define RGB_MATRIX_VAL_STEP 64
+#   define RGB_MATRIX_SPD_STEP 20
+#   define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_COLOR
 
 /* Disable the animations you don't want/need.  You will need to disable a good number of these    *
  * because they take up a lot of space.  Disable until you can successfully compile your firmware. */
-#    undef ENABLE_RGB_MATRIX_ALPHAS_MODS
```

Comment:
For what it's worth, all the `#undef ENABLE_RGB_MATRIX_...` can be removed, since they are not enabled to begin with.

Reply:
that's right, but I consider those definitions as a part of configuration.